### DISCUSSION TITLE
No beta for MacOS or Linux?

### DISCUSSION THREAD

**Discussion by Atque** [Stellarium/stellarium#2213] [General]
I noticed that the latest beta build has only been released for Windows 32 and 64 bits, but not for MacOS or Linux. Is this intentional?

(I have not figured out how to build from source on MacOS yet, and would like to test a few things in the master.)

**Comment by alex-w** [comment]
At this weekend I will publish binary snapshots for Linux and macOS too. Previously I did not publish both snapshots due changes in the requirement of infrastructure of building environment, because we switched to use C++17 standard.

**Comment by Atque** [comment]
@alex-w All right, great!

**Comment by t4saha** [comment]
For Ubuntu based Linux system binary files are available at launchpad PPA : https://code.launchpad.net/~stellarium/+recipe/stellarium-daily-git

**Reply by alex-w** [reply]
Yes, but I didn't prepare an AppImage ;-)

**Reply by t4saha** [reply]
Edge channel of Snap package is autoupdated in every 2 or 3 days or any repo is marged to master. It also support delta update.
